src/gimelstudio/core/node/property.py

```diff
@@ -20,7 +20,8 @@
 import gswidgetkit.foldpanelbar as fpbar
 from gswidgetkit import (NumberField, EVT_NUMBERFIELD,
                          Button, EVT_BUTTON, TextCtrl,
-                         Label, DropDown, EVT_DROPDOWN)
+                         Label, DropDown, EVT_DROPDOWN, 
```

Comment:
So, during testing I found a bug with the previous event for the colorpicker (which I've fixed in one of the recent commits to ``master``), so you'll want to import ``EVT_COLORPICKER_BUTTON`` here.